.travis.yml

```diff
@@ -6,20 +6,35 @@ matrix:
     - python: "2.7"
       env: TOXENV=py27 PYPI=true
       os: linux
+      before_install:
+        - sudo apt update
+        - sudo apt install python-dev libgirepository1.0-dev libcairo2-dev gir1.2-secret-1

```

Comment:
Very cool, do your tests actually run on a cloud machine with Linux? A bit surprised as I thought most distros require an X11 (e.g. Gnome Desktop) to be installed.